test/e2e/pods.go

```diff
@@ -828,39 +828,6 @@ var _ = Describe("Pods", func() {
 		}
 	})
 
-	It("should have their container restart back-off timer increase exponentially", func() {
```

Comment:
I think I am ok to remove this test since there are other tests to cover Container restart backoff and image backoff.